docs/notification_system.md

```diff
@@ -1,40 +1,45 @@
 # Notification System Overview
 
-A Notifier is an instance of a notification type (Email, Slack, Webhook, etc) with a name, description, and a defined configuration (A few examples: Username, password, server, recipients for the Email type.  Token and list of channels for Slack.  Url and Headers for webhooks)
+A Notifier is an instance of a notification type (Email, Slack, Webhook, etc.) with a name, description, and a defined configuration A few examples include:
 
-A Notification is a manifestation of the Notifier... for example, when a job fails a notification is sent using the configuration defined by the Notifier.
+* Username, password, server, recipients for the Email type.
+* Token and list of channels for Slack.
+* URL and Headers for webhooks.
 
-This PR implements the Notification system as outlined in the 3.0 Notifications spec.  At a high level the typical flow is:
+A Notification is a manifestation of the Notifier... for example, when a job fails, a notification is sent using the configuration defined by the Notifier.
```

Comment:
We should change `Notifier` to `NotificationTemplate`, I think "notifier" nomenclature is from the _old_ implementation before @matburt rewrote this.

Reply:
This goes for most references to "notifier" in this doc.